### ISSUE TITLE
`pip-sync` Travis Python dependencies.

### ISSUE THREAD

**Issue by esmail** [kobotoolbox/kpi#1152]

**Comment by esmail** [comment]
Easier after #1151.

**Comment by jnm** [comment]
Already done: https://github.com/kobotoolbox/kpi/commit/19b17e9c0e16cbe2a6d7b781c9000d68abaae244
